doc/input-output-reference/src/overview/group-plant-equipment.tex

```diff
@@ -6275,21 +6275,21 @@ \subsubsection{Inputs}\label{inputs-21-004}
 
 This is the name of the other node that connects the ice storage system to the plant loop. It is the outlet node to the ice storage component.
 
-\paragraph{Field: Discharging Curve Object Type}\label{field-discharging-curve-object-type}
+\paragraph{Field: Discharging Curve Variable Specifications}\label{field-discharging-curve-variable-specifications}
 
-The detailed ice storage model in EnergyPlus takes advantage of the Curve feature of the program. Currently, the only two allowed curve fit types for the detailed ice storage model are the QuadraticLinear and the CubicLinear curves. More information on this curve can be found in the section on Curves.
+The detailed ice storage model in EnergyPlus takes advantage of the Curve feature of the program.  The discharging curve can use any curve that has two independent variables.  This field controls which two parameters are the independent variables in modeling the performance of the detailed ice storage model during discharging. Currently, there are four different independent variable specifications: PercentChargedLMTD (where Percent Charged is the first independent variable and LMTD is the second independent variable), PercentDischargedLMTD (where Percent Disharged is the first independent variable and LMTD is the second independent variable), LMTDMassFlow (where LMTD is the first independent variable and Mass Flow rate is the second independent variable), and LMTDPercentCharged (where LMTD is the first independent variable and Percent Charged is the second independent variable). More information on curve types can be found in the section on Curves.  For additional information on how these curves are used in the Detailed Ice Storage model, please consult the Engineering Reference.
```

Comment:
This would read better as an itemized list. 
And (nit-picking) no need to say "Currently, there are." --> "There are four . . ."

Reply:
@mjwitte Fixed this...and correct the Percent typos here as well.